pkg/controller/podgroup_test.go

```diff
@@ -144,6 +153,13 @@ func Test_Run(t *testing.T) {
 
 			pgInformerFactory.Start(ctx.Done())
 			informerFactory.Start(ctx.Done())
+			// pod number scales down to 0
```

Comment:
We don't need to add this delete operation here. In this case, we can set 
```go
podNames: []string{}
```
Add this in line 143 or line 144
```go
if len(c.podsNames) != 0 { ... }
```

Reply:
done